server/src/dtos/asset-media.dto.ts

```diff
@@ -35,6 +46,28 @@ export class AssetMediaReplaceDto {
   [UploadFieldName.ASSET_DATA]!: any;
 }
 
+export class AssetMediaCreateDto extends AssetMediaBase {
+  @ValidateBoolean({ optional: true })
+  isFavorite?: boolean;
+
+  @ValidateBoolean({ optional: true })
+  isArchived?: boolean;
+
+  @ValidateBoolean({ optional: true })
+  isVisible?: boolean;
+
+  @ValidateBoolean({ optional: true })
+  isOffline?: boolean;
+
+  @ValidateUUID({ optional: true })
```

Comment:
I think we want to drop livePhotoVideoId - as the asset/livephoto should be uploaded separately.

Reply:
Yes it is, this is to link it when the second one is created.